packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -218,8 +227,15 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
 ///
 /// See also:
 ///
-///  * [VerticalMultiDragGestureRecognizer]
+///  * [HorizontalDragGestureRecognizer], for a similar recognizer but for
+///    horizontal movement.
+///  * [MultiDragGestureRecognizer], for a family of gesture recognizers that
+///    track each touch point independently.
 class VerticalDragGestureRecognizer extends DragGestureRecognizer {
+  /// Create a gesture recognizer for interactions intended for moving content
```

Comment:
This may be one of those cases where a comment can't improve on the name of the class very much. I don't think you really need to qualify the gesture's intent; maybe I'm using vertical gestures to zoom an image or change its opacity.